cmd/nimbess-cni.go

```diff
@@ -164,7 +202,54 @@ func cmdAdd(args *skel.CmdArgs) error {
 		CNIVersion: conf.CNIVersion,
 	}
 
-	// TODO: handle IPAM type and data
+	// Run the IPAM plugin and get back the config to apply
+	ipamRequest, err := ipam.ExecAdd(cniRequest.IpamType, args.StdinData)
```

Comment:
where are you setting this? I think it should come from Network Config right? I put it here:
https://github.com/nimbess/nimbess-agent/blob/32851b3d2e2ca452481a4aeccbc9e2fe00e8ef6a/k8s/nimbess.yaml#L16

Reply:
The Network Config doesn't specify any field for IPAM. Is it expected to be under the "Args" map?
https://github.com/nimbess/nimbess-agent/blob/master/pkg/proto/cni/cni.pb.go#L152